File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int tail= s.size()-1;
        int cnt = 0;
        while (tail>=0 and s[tail]==' ') tail--;
        while (tail>=0 and s[tail]!=' '){
            cnt++;
            tail--;
        }
        return cnt;
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        j = len(s)-1
        
        while (s[j]==' '):
            j-= 1
        
        cnt = 0
        while (j>=0 and s[j] != ' ' ):
            cnt += 1
            j -= 1
            
            
        return cnt